### PR TITLE
ENYO-2573: Updated Header to accommodate ApplicationCloseButton's…

### PR DESCRIPTION
**Pull request by Djspaceg**
… presence when included in Panels.

An update to the `adjustTitleWidth` method was necessary to factor in the margin from the close button's rules.

Enyo-DCO-1.1-Signed-off-by: Blake Stephens <blake.stephens@lge.com>

### CODE REVIEW

File: src/Header/Header.js [hunk 1]
```diff
@@ -638,13 +638,16 @@ module.exports = kind(
 			// Measure client area's width + 40px of spacing
 			client = this.$.client ? this.$.client.hasNode() : null,
 			clientWidth = client ? client.offsetWidth : null,
+			clientStyle = client ? (client.currentStyle || window.getComputedStyle(client)) : null,		// Originally by YuC @ http://stackoverflow.com/questions/349257/detecting-true-border-padding-and-margin-from-javascript
```
Comment (aarontam) [inline src/Header/Header.js]: Should we use our `dom.getComputedStyle` instead?

File: src/ListActions/ListActions.js [hunk 2]
```diff
@@ -172,6 +173,7 @@ var ListActionsDrawer = kind(
 		// Skip animation before render time
 		if (!this.$.client.hasNode()) { return; }
 		this.$.client.addRemoveClass('open', this.open);
+		this.doCustomizeCloseButton({showing: (this.open ? false : true)});
```
Comment (aarontam) [inline src/ListActions/ListActions.js]: Probably can simplify this to `{showing: !this.open}`.

File: src/Header/Header.js (hunk 1 above)
Comment (Djspaceg) [inline src/Header/Header.js]: Oops. Totally.

File: src/ListActions/ListActions.js (hunk 2 above)
Comment (Djspaceg) [inline src/ListActions/ListActions.js]: Yeah! Not sure why I did that super verbose way...

File: src/ListActions/ListActions.js [hunk 3]
```diff
@@ -400,7 +402,7 @@ var ListActions = module.exports = kind(
 	*/
 	drawerComponents: [
 		{name: 'drawer', spotlightDisabled: true, kind: ListActionsDrawer, classes: 'list-actions-drawer', onComplete: 'drawerAnimationEnd', open: false, spotlight: 'container', spotlightModal:true, components: [
-			{name: 'closeButton', kind: IconButton, icon: 'closex', classes: 'moon-popup-close moon-list-actions-close moon-neutral', ontap: 'expandContract', accessibilityLabel: $L('Close'), backgroundOpacity: 'transparent', defaultSpotlightDown:'listActions'},
+			{name: 'closeButton', kind: IconButton, icon: 'arrowsmallup', classes: 'moon-popup-close moon-list-actions-close moon-neutral', ontap: 'expandContract', accessibilityLabel: $L('Close'), backgroundOpacity: 'transparent', defaultSpotlightDown:'listActions'},
```
Comment (aarontam) [inline src/ListActions/ListActions.js]: More minor: curious if we had considered making this `arrowsmalldown` to match the close animation direction.

Comment (Djspaceg) [inline src/ListActions/ListActions.js]: Up was specifically chosen for the direction of the list actions animation direction.

Comment (aarontam) [inline src/ListActions/ListActions.js]: Ah never mind! DyamicPanelsSample is way outdated... :disappointed:

**Pull request merged**